ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_statistical.py

```diff
@@ -173,22 +173,47 @@ def test_quantile(
     )
 
 
+@st.composite
+def _dtype_and_value_without_row_all_same_value(draw):
+    dtype_and_x = draw(
+        helpers.dtype_and_values(
+            available_dtypes=["float32", "float64"],
+            num_arrays=2,
+            shared_dtype=True,
+            abs_smallest_val=1e-5,
+            min_num_dims=2,
+            max_num_dims=2,
+            min_dim_size=3,
+            max_dim_size=3,
+            min_value=-100,
+            max_value=100,
+            allow_nan=False,
+        ),
+    )
+
+    dtype, value = dtype_and_x
+    x = value[0]
+    y = value[1]
+    dim_1 = x.shape[0]
+    dim_2 = x.shape[1]
+    for i in range(dim_1):
+        if len(np.unique(x[i, :])) == 1:
+            x[i] = np.random.uniform(-100, 100, dim_2)
```

Comment:
The usage of `np.random` is prohibited, it doesn't provide a shrinking feature, which makes fixing a failing test harder. We should be using the strategies provided by both Ivy and Hypothesis to generate our data.

Reply:
Is that ok to replace all rows and columns which have the same value at every position here?